reports/California.md

```diff
@@ -70,6 +70,15 @@ The police are seen shooting at fleeing protestors and parked vehicles.
 * https://twitter.com/EarvinD/status/1268087279458775040
 * [abc7 report](https://abc7.com/deputies-shoot-pepper-balls-at-skateboarders-in-hollywood---video/6230652/)
 
+### LAPD officer beats multiple protesters that are filming them during a protest in Beverley Hills | May 30th
+
+An officer, who was identified as Officer Hwang by an eye-witness who filmed the incident on their chest mounted GoPro camera, is seen beating an individual in a black jacket who was filming the officers. After being beat by a baton, the individual in the black jacket falls, and the officer continues to beat them. The eye-witness runs over to shield and protect the individual from the officer, and was beaten by the officer from behind while helping the fallen individual escape. At the end of the video, they are hit by a less-than-lethal projectile.
```

Comment:
Please remove personal information of officers including first names and last names as we want to avoid doxxing anyone.

```suggestion
An officer is seen beating an individual in a black jacket who was filming the officers. After being beat by a baton, the individual in the black jacket falls, and the officer continues to beat them. The eye-witness runs over to shield and protect the individual from the officer, and was beaten by the officer from behind while helping the fallen individual escape. At the end of the video, they are hit by a less-than-lethal projectile.
```